src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,171 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user = new User(RegistrationServiceTestConstants.OVER_MIN_LOGIN,
+                RegistrationServiceTestConstants.OVER_MIN_PASSWORD,
+                RegistrationServiceTestConstants.OVER_MIN_AGE);
+    }
+
+    @Test
+    void register_existingUser_NotOk() {
+        Storage.people.add(user);
+        Assertions.assertThrows(RegistrationException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void register_userNull_NotOk() {
```

Comment:
```suggestion
    void register_userNull_notOk() {
```
please fix the same issue in all places